macros/first_value.sql

```diff
@@ -0,0 +1,19 @@
+{% macro first_value(first_value_field, partition_field, order_by_field, order="asc") -%}
+
+{{ adapter.dispatch('first_value', packages = fivetran_utils._get_utils_namespaces()) (first_value_field, partition_field, order_by_field, order="asc") }}
+
+{%- endmacro %}
+
+--Default first_value calculation
+{% macro default__first_value(first_value_field, partition_field, order_by_field, order="asc")  %}
+
+    first_value( {{ first_value_field }} ) over (partition by {{ partition_field }} order by {{ order_by_field }} {{ order }} )
```

Comment:
@fivetran-joemarkiewicz i think `first_value` by default includes nulls -- perhaps we should include that as an argument to pass this macro?

Reply:
It does include null values by default. I included the argument to the macros as we do want to ignore nulls for this output. Thanks!